### ISSUE TITLE
Archive?

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/theano-feedstock#39]
Given upstream has stopped maintaining this, wonder if we should go ahead and archive. Thoughts?

cc @conda-forge/core

**Comment by hmaarrfk** [comment]
thats more up to the maintainers of this feedstock. Not really core right? (i guess there is alit of overlap)

**Comment by nouiz** [comment]
Agreed that it is the feedstock maintainer that should decide this.
I still get some rare question on github issues.
Do you have any statistics about the usage of those feed?

More importantly, Theano was forked: https://github.com/aesara-devs/aesara
So maybe the maintainers of this feed want to make a feed for the fork?

**Comment by jakirkham** [comment]
> thats more up to the maintainers of this feedstock. Not really core right? (i guess there is alit of overlap)

I'm both. So +1

**Comment by jakirkham** [comment]
cc @conda-forge/theano

**Comment by ericmjl** [comment]
Yes, let's archive this feedstock. Theano has been superseded by Aesara.

**Comment by jakirkham** [comment]
> More importantly, Theano was forked: https://github.com/aesara-devs/aesara So maybe the maintainers of this feed want to make a feed for the fork?

Yep this was done

https://github.com/conda-forge/aesara-feedstock

**Comment by jakirkham** [comment]
Submitted PR ( https://github.com/conda-forge/admin-requests/pull/536 ) to archive this feedstock
